StreamChatClient_v3/WebSocketClient/WebSocketClient.swift

```diff
@@ -0,0 +1,10 @@
+//
+// WebSocketClient.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+class WebSocketClient {
+  init(currentUser: User) {}
```

Comment:
Basically `WebSocketClient` no need to know about the user. It needs only `URLRequest`.

Reply:
👍